Index: llvm-toolchain-snapshot_14~++20220128082735+c962b3fdf8dd/libcxx/include/atomic
===================================================================
--- llvm-toolchain-snapshot_14~++20220128082735+c962b3fdf8dd.orig/libcxx/include/atomic
+++ llvm-toolchain-snapshot_14~++20220128082735+c962b3fdf8dd/libcxx/include/atomic
@@ -2696,9 +2696,13 @@ typedef conditional<_LIBCPP_CONTENTION_L
 #endif
 
 #if !defined(_LIBCPP_NO_LOCK_FREE_TYPES)
+#ifndef NO_LOCK_FREE
 typedef atomic<__libcpp_signed_lock_free> atomic_signed_lock_free;
 typedef atomic<__libcpp_unsigned_lock_free> atomic_unsigned_lock_free;
 #endif
+#endif
+
+#undef NO_LOCK_FREE
 
 #define ATOMIC_FLAG_INIT {false}
 #define ATOMIC_VAR_INIT(__v) {__v}
